### ISSUE TITLE
Snapshot deploy failing with 401

### ISSUE THREAD

**Issue by kstafford3** [dropwizard/dropwizard-discovery#47]
```
[INFO] --- maven-deploy-plugin:2.8.2:deploy (default-deploy) @ dropwizard-discovery ---
[INFO] Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/io/dropwizard/modules/dropwizard-discovery/1.3.0-2-SNAPSHOT/maven-metadata.xml
[INFO] Uploading to sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/io/dropwizard/modules/dropwizard-discovery/1.3.0-2-SNAPSHOT/dropwizard-discovery-1.3.0-2-20190309.214455-1.jar
[INFO] Uploading to sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/io/dropwizard/modules/dropwizard-discovery/1.3.0-2-SNAPSHOT/dropwizard-discovery-1.3.0-2-20190309.214455-1.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.620 s
[INFO] Finished at: 2019-03-09T21:44:55Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project dropwizard-discovery: Failed to deploy artifacts: Could not transfer artifact io.dropwizard.modules:dropwizard-discovery:jar:1.3.0-2-20190309.214455-1 from/to sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots): Failed to transfer file https://oss.sonatype.org/content/repositories/snapshots/io/dropwizard/modules/dropwizard-discovery/1.3.0-2-SNAPSHOT/dropwizard-discovery-1.3.0-2-20190309.214455-1.jar with status code 401 -> [Help 1]
```

`CI_DEPLOY_USERNAME` and `CI_DEPLOY_PASSWORD` probably just need to be updated.

**Comment by kstafford3** [comment]
There's a problem with escaping the env var

**Comment by kstafford3** [comment]
Fixed by #50
